### PR TITLE
PVS-proxy: use MAC rather than truncated uuid in port name

### PR DESCRIPTION
**Pull request by robhoes**
Signed-off-by: Rob Hoes rob.hoes@citrix.com

### CODE REVIEW

File: ocaml/xapi/pvs_proxy_control.ml [hunk 1]
```diff
@@ -15,13 +15,14 @@
...
 open Threadext
+open Xstringext
 
 module D = Debug.Make(struct let name = "xapi_pvs_proxy_control" end)
 open D
 
 let proxy_port_name vif =
-  let uuid = String.sub vif.API.vIF_uuid 0 8 in
-  Printf.sprintf "pvs-%s" uuid
+  let mac = String.replace ":" "" vif.API.vIF_MAC in
```
Comment (jjd27) [inline ocaml/xapi/pvs_proxy_control.ml:26]: Cool. Maybe worth a comment explaining the maximum string length that you found for interface names, to explain the `String.replace`?

**Comment by jjd27** [comment]
Might be worth a note in the commit message stating that this guarantees that the interface name is unique to that VM, whereas previously there was a non-zero chance of a naming collision.

**Comment by robhoes** [comment]
@jjd27 Thanks, I've updated it.

**Comment by jjd27** [comment]
:shipit:

**Pull request merged**